lib/config/migration.spec.ts

```diff
@@ -497,7 +497,7 @@ describe('config/migration', () => {
       res = configMigration.migrateConfig(config);
       expect(res.isMigrated).toBe(true);
       expect(res.migratedConfig).toMatchObject({
-        extends: ['foo', ':unpublishSafe', 'bar', 'npm:unpublishSafe'],
+        extends: ['foo', 'npm:unpublishSafe', 'bar', 'npm:unpublishSafe'],
```

Comment:
I hope we're okay with duplication. Instead, we have to make implementation more complex.

Reply:
Ideally we should filter out duplicates, especially as we plan to turn this migration into a PR for users soon. Can we check that it's not already in the array before adding it?

Reply:
Okay, we're not aiming to fix user-defined duplicates, but now we won't make new ones